test/it/linked-object.js

```diff
@@ -0,0 +1,165 @@
+const expect = require('chai').expect;
+const deepcopy = require('deep-copy');
+const okta = require('../../src');
+const models = require('../../src/models');
+const Collection = require('../../src/collection');
+const utils = require('../utils');
+const mockLinkedObject = require('./mocks/linked-object.json');
+const mockUser = require('./mocks/user-without-credentials.json');
+let orgUrl = process.env.OKTA_CLIENT_ORGURL;
+
+if (process.env.OKTA_USE_MOCK) {
+  orgUrl = `${orgUrl}/linked-object`;
+}
+
+const client = new okta.Client({
+  orgUrl: orgUrl,
+  token: process.env.OKTA_CLIENT_TOKEN,
+  requestExecutor: new okta.DefaultRequestExecutor()
+});
+
+describe('Linked Object API', () => {
+  describe('Link Definition Operations', () => {
+    let linkedObject;
+    describe('Add linked object definition', () => {
+      afterEach(async () => {
+        await linkedObject.delete(linkedObject.primary.name);
+      });
+
+      it('should return instance of LinkedObject model', async () => {
+        linkedObject = await client.addLinkedObjectDefinition(mockLinkedObject);
+        expect(linkedObject).to.be.instanceOf(models.LinkedObject);
+      });
+
+      it('should match name from request', async () => {
+        linkedObject = await client.addLinkedObjectDefinition(mockLinkedObject);
+        expect(linkedObject.primary.name).to.equal(mockLinkedObject.primary.name);
+        expect(linkedObject.associated.name).to.equal(mockLinkedObject.associated.name);
+      });
+    });
+
+    describe('Get a linked object definition by name', () => {
+      let linkedObjectFromGet;
+      beforeEach(async () => {
+        linkedObject = await client.addLinkedObjectDefinition(mockLinkedObject);
+      });
+      afterEach(async () => {
+        await linkedObject.delete(linkedObject.primary.name);
+      });
+
+      it('should return LinkedObject by primary name', async () => {
+        linkedObjectFromGet = await client.getLinkedObjectDefinition(linkedObject.primary.name);
+        expect(linkedObjectFromGet).to.be.instanceOf(models.LinkedObject);
+        expect(linkedObjectFromGet.primary.name).to.equal(mockLinkedObject.primary.name);
+      });
+
+      it('should return LinkedObject by associated name', async () => {
```

Comment:
These two test cases match two different scenarios from [D.O.C](https://developer.okta.com/docs/reference/api/linked-objects/#get-a-linked-object-definition-by-name), I would prefer keeping them separate instead of adding comments. The tradeoff is it will take the total running time a bit longer...